src/NHSD.BuyingCatalogue.Ordering.Api/Services/CompleteOrder/CompleteOrderService.cs

```diff
@@ -45,13 +45,16 @@ public async Task<Result> CompleteAsync(CompleteOrderRequest request)
 
             await _orderRepository.UpdateOrderAsync(order);
 
-            await using var stream = new MemoryStream();
-            await _createPurchasingDocumentService.CreateDocumentAsync(stream, order);
-            stream.Position = 0;
-
-            var emailMessage = _purchasingSettings.EmailMessage;
-            emailMessage.Attachments.Add(new EmailAttachment("CompletedOrder.csv", stream));
-            await _emailService.SendEmailAsync(emailMessage);
+            if (order.FundingSourceOnlyGMS != null && (bool)order.FundingSourceOnlyGMS)
```

Comment:
Just a suggestion... can use the method GetValueOrDefault no need to check for null.